vip_spec.xsd

```diff
@@ -630,6 +652,14 @@ reference elements for polling location
       <xs:enumeration value="super-majority" />
     </xs:restriction>
   </xs:simpleType>
+
+  <xs:simpleType name="VoterServiceType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="absentee-ballots" />
+      <xs:enumeration value="polling-places" />
+      <xs:enumeration value="voter-registration" />
```

Comment:
Make sure to put an "other" type in here. We should only check the "OtherType" field if "other" is present in the "Type" field.